conda-store-server/conda_store_server/server/views/api.py

```diff
@@ -564,6 +564,8 @@ async def api_post_specification(
         specification = schema.CondaSpecification.parse_obj(specification)
     except yaml.error.YAMLError:
         raise HTTPException(status_code=400, detail="Unable to parse. Invalid YAML")
+    except ValueError as e:
```

Comment:
You should use a custom exception here. Something else could raise a ValueError which wouldn't have a list in its args, and that probably shouldn't be converted to an HTTPException anyway.

Reply:
Done.